packages/strapi-plugin-i18n/admin/src/components/ModalCreate/index.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Modal,
+  ModalHeader,
+  HeaderModal,
+  HeaderModalTitle,
+  ModalFooter,
+  ModalForm,
+  Tabs,
+  TabsNav,
+  Tab,
+  TabsPanel,
+  TabPanel,
+} from 'strapi-helper-plugin';
+import { useIntl } from 'react-intl';
+import { Button } from '@buffetjs/core';
+import { Formik } from 'formik';
+import { object, string } from 'yup';
+import { getTrad } from '../../utils';
+
+const ModalCreate = ({ onClose, isOpened }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Modal isOpen={isOpened} onToggle={onClose}>
+      <HeaderModal>
+        <ModalHeader
+          headerBreadcrumbs={[formatMessage({ id: getTrad('Settings.list.actions.add') })]}
+        />
+      </HeaderModal>
+
+      <Formik
+        initialValues={{ displayName: '' }}
+        onSubmit={() => null}
+        validationSchema={object().shape({
+          displayName: string().max(50, 'Settings.locales.modal.create.locales.displayName.error'),
+        })}
+      >
+        {({ handleSubmit, errors }) => (
+          <form onSubmit={handleSubmit}>
+            <div className="container-fluid">
+              <div className="container-fluid">
+                <HeaderModalTitle
+                  style={{
+                    fontSize: '1.8rem',
+                    height: '65px',
+                    fontWeight: 'bold',
+                    alignItems: 'center',
+                    marginBottom: '-39px',
+                    paddingTop: '16px',
+                  }}
+                >
+                  {formatMessage({
+                    id: getTrad('Settings.locales.modal.title'),
+                  })}
+                </HeaderModalTitle>
+
+                <ModalForm>
+                  <TabsNav
```

Comment:
Maybe you can create a component for the ModalHeader and the Tabs so we can reuse the same one for both the edit and the create logic. What do you think?

Reply:
I'm not 100% convinced it's necessary to extract the Tabs component: what would be the goal and the shape of the new component? Don't you think it will close us from the advents of the compound component?

Concerning the ModalHeader, I think it's the same shape in both edit and create 🤔 . Do you have a specific idea in mind? 😊

Reply:
> what would be the goal and the shape of the new component? 

The goal would be to be able to easily update both the ModalCreate and the ModalEdit, for instance, if we were to add an icon to the header of the modal like in the CTB we would need to update only one component.

This component does not need to stand in the helper plugin but instead in the settings, the shape would then be:
```
<HeaderModalTitle>
  <ModalForm>
    <Tabs><Tab /> <Tab /></Tabs> // This is static and common to both the edit and create modals we don't need to duplicate the logic
   <TabsPanel>{children}</Tabs> 
  </ModalForm>
</HeaderModalTitle>
```